packages/cadl-ranch-specs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure-tools/cadl-ranch-specs",
-  "version": "0.39.5",
+  "version": "1.0.0",
```

Comment:
Interesting - it looks like this did a major version bump.

Reply:
I didn't look at it closely because it was fully auto-generated